### PR TITLE
fix(HMSPROV-407): enable cloudwatch in clowder

### PR DESCRIPTION
**Pull request by lzap**
Looks like everything is ready to enable cloudwatch API on stage, this is the only change that is needed to export logs via the API.

To get access to ELK/CW on AWS open the [HMSPROV-407](https://issues.redhat.com/browse/HMSPROV-407) ticket and follow the instructions to setup permissions for you account. You will need a GPG public key for a password to be emailed to you.

Testing: Not sure how to test this in ephemeral, but ideally let us merge this sometime in the morning hours so we can check stage. I am still waiting for my permissions to be approved.

### CODE REVIEW

**Comment by lzap** [comment]
Rebased to solve panic in ephemeral, added small logging enhancements in regard to reservation id logging.

**Comment by lzap** [comment]
Failed: `response` not updated with instances

This is because of https://issues.redhat.com/browse/HMSPROV-414

I am going to merge this since cloudwatch/kibana will make investigation much easier.

File: internal/services/aws_reservation_service.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 )
 
 func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
-	logger := ctxval.Logger(r.Context())
+	logger := *ctxval.Logger(r.Context())
```
Comment (adiabramovitch) [inline internal/services/aws_reservation_service.go:22]: Just wondering, we didn't use the '*' until now, why did you add it? 
it seems like the compiler knows to reference it like the logger object and not as a pointer.

**Review by adiabramovitch — APPROVED** [review]
Just a small question for general knowledge

File: internal/services/aws_reservation_service.go [hunk 2]
```diff
@@ -68,6 +68,7 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 		renderError(w, r, payloads.NewDAOError(r.Context(), "create reservation", err))
 		return
 	}
+	logger = logger.With().Int64("reservation_id", reservation.ID).Logger()
```
Comment (lzap) [inline internal/services/aws_reservation_service.go:71]: It is because of this line, I would have to introduce a helper variable here.

File: internal/services/aws_reservation_service.go (hunk 1 above)
Comment (lzap) [reply internal/services/aws_reservation_service.go:22]: That is indeed suspicions, I did this on purpose (see my comment below).

It would have been dangerous, but I know for fact that this function never returns `nil`: https://github.com/RHEnVision/provisioning-backend/blob/main/internal/ctxval/context_values.go#L34
